msteams-platform/get-started/glossary.md

```diff
@@ -260,11 +259,13 @@ Common terms and definitions used in Microsoft Teams developer documentation.
 | [Thread discussion](../tabs/design/tabs.md#thread-discussion) | A conversation posted on a channel or chat between users. <br>**See also** [Conversation](#c); [Channel](#c) |
 | [Teams](../overview.md) | Microsoft Teams is the ultimate message app for your organization. It's a workspace for real-time collaboration and communication, meetings, file and app sharing. |
 | [Teams AI library](../bots/how-to/teams-conversational-ai/teams-conversation-ai-overview.md) | A Teams-centric interface to GPT-based common language models and user intent engines. You can take on complex and expensive tasks of writing and maintaining conversational bot logic to integrate with Large Language Models (LLMs).|
+| [Teams app capabilities](../concepts/design/map-use-cases.md#app-capabilities-mapped-to-features)| The Teams app capabilities are the core functionalities that you can build in your app. They're also called entry or extension points because they enable integration and interaction. <br> **See also**: [App scope](#a)|
 | [Teams identity](../tabs/how-to/authentication/tab-sso-overview.md) | The Microsoft account or Microsoft 365 account of an app user that is used to log in to Teams client, web, or mobile app. |
 | [Teams identity](../tabs/how-to/authentication/tab-sso-overview.md) | The Microsoft account or Microsoft 365 account of an app user that is used to sign in to Teams client, web, or mobile app. |
 | [Teams Toolkit](../toolkit/teams-toolkit-fundamentals.md) | The Microsoft Teams Toolkit enables you to create custom Teams apps directly within the VS Code environment.  |
 | [Teams Toolkit CLI](../toolkit/Teams-Toolkit-CLI.md) | Teams Toolkit CLI is a text-based command line interface that accelerates Teams application development. It's also called Teamsapp CLI.|
 | [TeamsFx SDK](../toolkit/teamsfx-sdk.md) | TeamsFx SDK is pre-configured in a scaffolded project using TeamsFx toolkit or CLI. |
+| [TeamsJS capability](../m365-apps/teamsjs-support-m365.md)| The TeamsJS capability is a logical grouping of functionality that is atomatically implemented by a Microsoft 365 host. |
```

Comment:
Typo: atomatically 

This seems misleading: TJS capabilities are not automatically implemented by M365 hosts, and thus there's a lot of discrepancies in support between Outlook/Harmony versus Teams. How about something like "is a logical grouping of functionality supported across one or more Microsoft 365 application hosts."

Reply:
Updated as suggested